jsx/Form.js

```diff
@@ -1365,6 +1365,57 @@ StaticElement.defaultProps = {
   text: null,
 };
 
+/**
+ * Email element component.
+ * Used to displays plain/formatted email input as part of a form
+ *
+ * To pass a formatted text, you need to wrap it in a single parent element.
+ * Example usage:
+ *
+ * ```
+ * let myText = (<span>This is my <b>text</b></span>);
+ * <EmailElement
+ *    text={myText}
+ *    label={note}
+ * />
+ * ```
+ */
+class EmailElement extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div className="row form-group">
+        <label className="col-sm-3 control-label">
+          {this.props.label}
+        </label>
+        <div className="col-sm-9">
+          <input className="form-control-static" type={'email'} id={this.props.id}>
```

Comment:
i think just 'email' without the curlies is sufficient no?

Reply:
@zaliqarosli My IDE does that by default, I don't think it matters but removed.